src/app/(dashboard)/requirements/[reqId]/page.tsx

```
@@ -0,0 +1,33 @@
+  // TODO: Supabase セッションから teamId を取得する（#006 マージ後に置き換え）
+  const { teamId = '' } = await searchParams
+
+  try {
+    const requirement = await getRequirement(teamId, reqId)
+    return (
+      <div className="flex h-screen bg-[#F2F3F0]">
+        {/* TODO: ダッシュボード Sidebar を追加（#006 マージ後） */}
+        <main className="flex-1 min-w-0 overflow-hidden">
+          <RequirementsView requirement={requirement} />
+        </main>
+      </div>
```

ESLint

[error] Error: Avoid constructing JSX within try/catch

React does not immediately render components when JSX is rendered, so any errors from this component will not be caught by the try/catch. To catch errors in rendering a given component, wrap that component in an error boundary. (https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary).

/home/jailuser/git/src/app/(dashboard)/requirements/[reqId]/page.tsx:23:7
  21 |     const requirement = await getRequirement(teamId, reqId)
  22 |     return (
> 23 |       <div className="flex h-screen bg-[`#F2F3F0`]">
     |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 24 |         {/* TODO: ダッシュボード Sidebar を追加（#006 マージ後） */}
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 25 |         <main className="flex-1 min-w-0 overflow-hidden">
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 26 |           <RequirementsView requirement={requirement} />
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 27 |         </main>
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 28 |       </div>
     | ^^^^^^^^^^^^^ Avoid constructing JSX within try/catch
  29 |     )
  30 |   } catch {
  31 |     notFound()

(react-hooks/error-boundaries)